test/services/return-requirements/fetch-licence-purposes.service.test.js

```diff
@@ -14,9 +14,9 @@ const LicenceVersionPurposeHelper = require('../../support/helpers/licence-versi
 const PurposeHelper = require('../../support/helpers/purpose.helper.js')
 
 // Thing under test
-const FetchPurposesService = require('../../../app/services/return-requirements/fetch-purposes.service.js')
+const FetchPurposesService = require('../../../app/services/return-requirements/fetch-licence-purposes.service.js')
 
-describe('Return Requirements - Fetch Purposes service', () => {
+describe('Return Requirements - Fetch Purposes for a given licence service', () => {
```

Comment:
Can't disagree the title is more meaningful. But it breaks our convention of using the module name.

```suggestion
describe('Return Requirements - Fetch Licence Purposes service', () => {
```